include/mysql/service_wsrep.h

```diff
@@ -80,7 +80,7 @@ extern struct wsrep_service_st {
   void                        (*wsrep_handle_SR_rollback_func)(MYSQL_THD BF_thd, MYSQL_THD victim_thd);
   my_bool                     (*wsrep_thd_skip_locking_func)(const MYSQL_THD thd);
   const char*                 (*wsrep_get_sr_table_name_func)();
-  my_bool                     (*wsrep_get_debug_func)();
+  my_bool                     (*wsrep_debug_enabled)();
```

Comment:
I think it would be better to keep the original name of the function in order to keep the patch footprint small and reduce chance of merge conflicts in merges from earlier versions.